### PR TITLE
Completed prep-html-links exercise

### PR DESCRIPTION
**Pull request by gabriellerramos**
https://user-images.githubusercontent.com/113288280/190928322-6e36d913-a509-4588-9b5b-782c6b7c3bb9.mov

### CODE REVIEW

File: prep-html-links/contact-us/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<head>
+  <title>Contact Us</title>
+</head>
+  <h1>Contact Us</h1>
+  <a href="/prep-html-link/index.html">Click here to go to the home page</a>
+  <link href="prep-html-link/index" rel="contact-us/index">
```
Comment (thebearingedge) [inline prep-html-links/contact-us/contact-us.html]: Hi Gabrielle, be sure to put the visible content of your page into a `body` element and the `head` and `body` both belong in an `html` element, as outlined in the `prep-html-skeleton` exercise.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by gabriellerramos** [comment]
Screen Shot 2022-09-20 at 7 43 18 PM

File: prep-html-links/contact-us/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="index.html">Click here to go to the home page</a>
+</body>
```
Comment (thebearingedge) [inline prep-html-links/contact-us/contact-us.html]: Clicking this link should go to the home page, so you'll need to change your `href` value to `"../index.html"`

File: prep-html-links/index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="contact-us"> Click here to go to the contact page </a>
```
Comment (thebearingedge) [inline prep-html-links/index.html]: Clicking this link should go to the `contact-us.html` file inside of the `contact-us` folder, so you'll need to change your `href` value to `"contact-us/contact-us.html"`

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by gabriellerramos** [comment]
Screen Shot 2022-09-20 at 7 43 18 PM

File: prep-html-links/contact-us/contact-us.html (hunk 2 above)
Comment (thebearingedge) [reply prep-html-links/contact-us/contact-us.html]: This one will also need to get changed ☝️

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-html-links/index.html [hunk 4]
```diff
@@ -5,6 +5,6 @@
 </head>
 <body>
   <h1>Home page</h1>
-  <a href="contact-us/contact-us.html"> Click here to go to the contact page </a>
+  <a href="contact-us/index.html"> Click here to go to the contact page </a>
```
Comment (thebearingedge) [inline prep-html-links/index.html]: Unfortunately, this link won't work because there is no `index.html` in the `contact-us` directory.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by gabriellerramos** [comment]
Screen Shot 2022-10-05 at 9 44 18 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**